### PR TITLE
Added discovery 5.0 release

### PR DESCRIPTION
**Pull request by lzap**

### CODE REVIEW

File: plugins/foreman_discovery/5.0/index.md [hunk 1]
```diff
@@ -0,0 +1,1102 @@
+---
+layout: plugin
+pluginname: foreman_discovery
+title: Foreman Discovery 4.1 Manual
```
Comment (domcleal) [inline plugins/foreman_discovery/5.0/index.md]: 5.0

Comment (lzap) [inline plugins/foreman_discovery/5.0/index.md]: Damn, thanks. FYI no changes other than the changelog.

**Comment by lzap** [comment]
If that's all... reabased. Thanks!

**Comment by domcleal** [comment]
Merged, thanks @lzap.

**Pull request merged**